### PR TITLE
Initial commit of Megalinter configuration

### PR DESCRIPTION
**Pull request by wesley-dean-gsa**
This adds Megalinter configuration and a workflow to invoke upon commit push.  Note: fixes are Not automatically applied at this time.

The workflow is pinned to Megalinter v5 due to some DevSkim hiccups.

This is related to #14

### CODE REVIEW

File: .mega-linter.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+# don't use Mega-Linter to test the reports Mega-Linter created
+ADDITIONAL_EXCLUDED_DIRECTORIES: [ report, megalinter-reports ]
```
Comment (cantsin) [inline .mega-linter.yml:3]: Ironically, perhaps, MegaLinter doesn't like this file: there's a few `12:11 [brackets] too many spaces inside brackets` warnings here. Can we fix real quick?

**Review by cantsin — CHANGES_REQUESTED** [review]
Slight change requested! (Fixing megalinter's warnings for this very PR!)

File: .mega-linter.yml (hunk 1 above)
Comment (wesley-dean-gsa) [reply .mega-linter.yml:3]: You know what's also funny?  Megalinter, when `APPLY_FIXES` is set, will automatically get stuff like that.

But yes, absolutely, I'll fix it right now.

**Comment by wesley-dean-gsa** [comment]
@cantsin changes applied

**Comment by wesley-dean-gsa** [comment]
converted back to a draft; waiting for a PAT to allow Megalinter to push changes back to the repo

**Comment by wesley-dean-gsa** [comment]
note: `APPLY_FIXES` has been enabled (despite the PR's original text)

**Review by cantsin — APPROVED** [review]
LGTM!

**Pull request merged**